src/screens/FeedsManager/FeedsManagerCard.test.tsx

```diff
@@ -1,20 +1,24 @@
 import * as React from 'react'
 
+import userEvent from '@testing-library/user-event'
 import { Route } from 'react-router-dom'
 import { renderWithRouter, screen } from 'support/test-utils'
-import userEvent from '@testing-library/user-event'
 
-import { FeedsManagerCard } from './FeedsManagerCard'
-import { buildFeedsManagerFields } from 'support/factories/gql/fetchFeedsManagersWithProposals'
 import { shortenHex } from 'src/utils/shortenHex'
+import { buildFeedsManagerFields } from 'support/factories/gql/fetchFeedsManagersWithProposals'
+import { FeedsManagerCard } from './FeedsManagerCard'
 
 const { getByRole, queryByText } = screen
 
 function renderComponent(manager: FeedsManagerFields) {
   renderWithRouter(
     <>
       <Route path="/">
-        <FeedsManagerCard manager={manager} />
+        <FeedsManagerCard
+          manager={manager}
+          onDisable={() => {}}
+          onEnable={() => {}}
```

Comment:
Like how we have the test `navigate to edit`, do u think we could test clicking disable and enable will call onDisable and onEnable?